app/models/work_version.rb

```diff
@@ -60,6 +66,10 @@ def clean_abstract
     Rails.logger.info { "Null character stripped from abstract for work_version id #{id}." }
   end
 
+  def globus
+    upload_type == 'globus'
+  end
+
```

Comment:
helper method so that various other parts of the existing codebase that check to see if a work is using globus or not will work without further change

we can add additional methods as needed for the other types in future PRs